color_refiment_helper.py

```diff
@@ -134,3 +134,10 @@ def get_unit_coloring(graph: "Graph"):
         coloring.set(0, v)
     debug('Init coloring ', coloring)
     return coloring
+
+
+def get_coloring(graph: "Graph")->Coloring:
```

Comment:
Overbodig. Hier hebben we `initialize_coloring` en `get_unit_coloring` voor. Nb de coloring class zet de colornum.

Reply:
Nope, dit doet wat anders. Dit maakt een coloring met de huidige colornums. initialize doet op degree en unit geeft het 1 kleur.